packages/cadl-ranch-specs/http/collectionFormat/main.tsp

```diff
@@ -1,88 +0,0 @@
-import "@typespec/http";
```

Comment:
why is this test case being removed ?

Reply:
that was accidental, have aded it back in a later pr